godtools/App/Services/Renderer/Views/MobileContent/Views/Pages/MobileContentPagesViewModel.swift

```diff
@@ -43,6 +43,8 @@ class MobileContentPagesViewModel: NSObject, ObservableObject {
     let pageNavigationEventSignal: SignalValue<MobileContentPagesNavigationEvent> = SignalValue()
     let incrementUserCounterUseCase: IncrementUserCounterUseCase
     
+    var toolSettingsObserver: ToolSettingsObserver?
```

Comment:
Hey @rachaelblue I'd say make this ```private(set) var``` since this is set in the ```setup``` method of the parent class.